feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/handlers/location/motivate/DefaultMotivateUseGpsHandler.kt

```diff
@@ -0,0 +1,52 @@
+package com.epmedu.animeal.home.presentation.viewmodel.handlers.location.motivate
```

Comment:
I have a question regarding logic location. Although the map and its permissions requests are currently shown at home, if those are needed elsewhere ( i.e. profile or login) it could become difficult to decouple it, it would be possible to move this to the shared module?

Reply:
Good question, since it's not a permission request and more about motivating the user to use the GPS for location. And since it uses before building the route, I thought that it directly the Home Page UI. That's why I decided to place that logic here.